spec/serializers/postgres_spec.rb

```diff
@@ -3,61 +3,97 @@
 require_relative "../spec_helper"
 
 RSpec.describe Serializers::Postgres do
-  let(:pg) { PostgresResource.new(name: "pg-name", location_id: Location::HETZNER_FSN1_ID).tap { it.id = "69c0f4cd-99c1-8ed0-acfe-7b013ce2fa0b" } }
+  let(:project) { Project.create(name: "pg-test-project") }
+  let(:location_id) { Location::HETZNER_FSN1_ID }
+  let(:location) { Location[location_id] }
+  let(:timeline) { PostgresTimeline.create(location_id: location_id) }
+  let(:private_subnet) {
+    PrivateSubnet.create(
+      name: "pg-subnet", project_id: project.id, location_id: location_id,
+      net4: "172.0.0.0/26", net6: "fdfa:b5aa:14a3:4a3d::/64"
+    )
+  }
+  let(:time) {
+    t = Time.now
+    allow(Time).to receive(:now).and_return(t)
```

Comment:
The only 3 specs where this method is called used `expect` , so we should probably use the same here.

```suggestion
    expect(Time).to receive(:now).and_return(t).at_least(:once)
```